docs/user-guide/ze-profiles.md

```diff
@@ -0,0 +1,67 @@
+# Working with Zowe Explorer profiles
+
+You must have a `zosmf` compatible profile before you can use Zowe Explorer. You can set up a profile to retain your credentials, host, and port name. In addition, you can create multiple profiles and use them simultaneously.
+
+**Follow these steps:**
+
+1. Navigate to the explorer tree.
+2. Click the **+** button next to the **DATA SETS**, **USS** or **JOBS** bar.
+
+   **Note:** If you already have a profile, select it from the drop-down menu.
+
+3. Select the **Create a New Connection to z/OS** option.
+
+   **Note:** When you create a new profile, user name and password fields are optional. However, the system will prompt you to specify your credentials when you use the new profile for the first time.
+
+4. Follow the instructions, and enter all required information to complete the profile creation.
+
+![New Connection](../images/ze/ZE-newProfiles.gif?raw=true "New Connection")
+<br /><br />
+
+You successfully created a Zowe CLI `zosmf` profile. Now you can use all the functionalities of the extension.
+
+If you need to edit a profile, click the **Update Profile** button next to the corresponding profile.
+
+![Edit a Profile](../images/ze/ZE-edit-ze-profile.gif?raw=true "Edit a Profile")
+<br /><br />
+
+In addition, you to hide a profile from the explorer tree, and permanently delete a profile. When you delete your profile permanently, the extension erases the profile from the `.zowe` folder. To hide or delete a profile, right-click the profile and choose one of the respective options from the list.
+
+## Enabling Secure Credential Store with Zowe Explorer
+
+You can enable a high level of security for you credentials, using the Secure Credential Store (SCS) plug-in in the Zowe Explorer extension. By default, your credentials are stored in plain text, which increases the vulnerability of your system. The SCS plug-in lets you store your credentials securely and avoid an unnecessary information leakage.
+
+Activate the SCS plug-in in Zowe Explorer.
+
+**Follow these steps:**
+
+1. Open Zowe Explorer.
+2. Navigate to the VSCode settings.
+3. Open Zowe Explorer Settings.
+4. Add the **Zowe-Plugin** value to the `Zowe Security: Credential Key` entry field.
+5. Restart VSCode.
+6. Create a profile.
+
+Your Zowe Explorer credentials are now stored securely.
+
+### For Zowe CLI users
```

Comment:
So you don't need CLI installed, but IF you do use CLI then you need to be careful about how you use it in conjuction with the explorer. Right? 

Just making sure that the previous section makes sense w/out CLI.

Reply:
Yes, something like that. Although 'careful' sounds a bit vague. Let me elaborate a bit :)
Case 1. If you want to just activate SCS in ZE, you won't need to have CLI installed on your computer. 
Case 2. If you do use CLI and want to activate SCS in ZE, then you need to install the SCS plug-in for CL > run the zowe scs u command in CLI > activate SCS in ZE. Activating SCS in ZE w/o installing SCS plug-in (and running the command) will lead to issues with your existing profiles.